src/main/kotlin/lotto/model/winning/WinningCondition.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.model.winning
+
+import lotto.model.LottoTicket
+import lotto.model.number.LottoNumber
+import lotto.model.number.LottoNumbers
+
+data class WinningCondition(val winningNumbers: LottoNumbers, val bonusNumber: LottoNumber) {
+    init {
+        require(!winningNumbers.isMatch(bonusNumber)) { "보너스 번호와 당첨 번호는 중복될 수 없습니다!" }
+    }
+
+    fun check(lottoTicket: LottoTicket): Pair<Int, Boolean> {
+        val winningCount = lottoTicket.countMatch(winningNumbers)
+        val bonusMatch = lottoTicket.isMatch(bonusNumber)
+
+        return Pair(winningCount, bonusMatch)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return winningCount to bonusMatch
```